handlers/deploy.go

```diff
@@ -53,7 +57,7 @@ func MakeDeployHandler(functionNamespace string, clientset *kubernetes.Clientset
 			return
 		}
 
-		deploymentSpec := makeDeploymentSpec(request)
+		deploymentSpec := makeDeploymentSpec(request, config.EnableFunctionReadinessProbe)
```

Comment:
Let's pass the whole config object to each endpoint.

Reply:
@alexellis Sure, but it's not clear to me why that's preferred. Would you mind explaining?